components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementServiceImpl.java

```diff
@@ -1344,6 +1366,40 @@ public ImportResponse importSPApplication(ServiceProvider serviceProvider, Strin
         return importResponse;
     }
 
+    private void validateResourceId(String resourceId, String tenantDomain)
+            throws IdentityApplicationManagementException {
+
+        if (StringUtils.isEmpty(resourceId)) {
+            String message = "Invalid argument: Authenticator resource ID value is empty";
```

Comment:
Let's add more context here like tenant domain details

Reply:
The same PR is merged to master branch. We have created this PR since the auto PR creation will failed. I will create a separate PR for the master branch.